pkg/storage/store.go

```diff
@@ -3752,9 +3752,8 @@ func (s *Store) updateReplicationGauges(ctx context.Context) error {
 			// If this replica is the highest replica ID, it does the counting.
 			if highestIdx != -1 && desc.Replicas[highestIdx].StoreID == s.StoreID() {
 				rangeCount++
-				// If a quorum of replicas are live, consider this range available.
-				if liveReplicas > computeQuorum(len(desc.Replicas)) {
```

Comment:
Updated the admin UI and CLI.

Changing the metric is a good™ change because it's more consistent with the under-replicated ranges metric, and also a more useful metric in its own right when displayed in the admin UI.